src/databricks/labs/ucx/hive_metastore/grants.py

```diff
@@ -715,3 +716,57 @@ def _get_location_name(self, location_url: str):
             if location.url == location_url:
                 return location.name
         return None
+
+
+class MigrateGrants:
+    def __init__(
+        self,
+        sql_backend: SqlBackend,
+        group_manager: GroupManager,
+        grant_loaders: list[Callable[[], Iterable[Grant]]],
+    ):
+        self._sql_backend = sql_backend
+        self._group_manager = group_manager
+        self._grant_loaders = grant_loaders
+
+    def apply(self, src: Table, uc_table_key: str) -> bool:
+        for grant in self._match_grants(src):
+            acl_migrate_sql = grant.uc_grant_sql(src.kind, uc_table_key)
+            if acl_migrate_sql is None:
+                logger.warning(f"Cannot identify UC grant for {src.kind} {uc_table_key}. Skipping.")
+                continue
+            logger.debug(f"Migrating acls on {uc_table_key} using SQL query: {acl_migrate_sql}")
+            try:
+                self._sql_backend.execute(acl_migrate_sql)
+            except DatabricksError as e:
+                logger.warning(f"Failed to migrate ACL for {src.key} to {uc_table_key}: {e}")
+        return True
+
+    @cached_property
+    def _workspace_to_account_names(self) -> dict[str, str]:
```

Comment:
Please rename this property so that it is clear it is mapping groups, that confused me at first when reading `self._workspace_to_account_names.get(grant.principal)` below